aps/utils/methods.py

```diff
@@ -342,12 +342,9 @@ def calc_average(
     Output:
             average  - average value
     """
+    average, number_of_elements = np.average(values[cell_index_defined], returned=True)
     if debug_level >= Debug.VERY_VERBOSE:
-        average, number_of_elements = np.average(
-            values[cell_index_defined], returned=True
-        )
         print(f'--- Average: {average}  Number of elements: {number_of_elements}')
-        return average
     return np.average(values[cell_index_defined])
```

Comment:
Now that you do this call anyways, you don't need to `return np.average(...`, you can just return `average`, right?

Reply:
Yes